lstchain/datachecks/dl1_checker.py

```diff
@@ -244,6 +242,12 @@ def process_dl1_file(filename, bins, trigger_source='trigger_type'):
         # trigger type tags are not reliable. We first identify flatfield events
         # by their looks.
         image = image_table.col('image')
+        # define criteria for detecting flatfield events, since as of 20200418
```

Comment:
why moving the `ff_min_pixel_charge_median` and `ff_max_pixel_charge_stddev` inside the table opening?

Reply:
Same here, it was suggested by the IDE: "move assignment closer to their usage".

Both changes can be reverted if necessary.